packages/server/graphql/types/AtlassianIntegration.ts

```diff
@@ -182,20 +184,18 @@ const AtlassianIntegration = new GraphQLObjectType<any, GQLContext>({
       type: GraphQLNonNull(GraphQLList(GraphQLNonNull(JiraSearchQuery))),
       description:
         'the list of suggested search queries, sorted by most recent. Guaranteed to be < 60 days old',
-      resolve: async ({id: atlassianAuthId, jiraSearchQueries}: AtlassianAuth) => {
+      resolve: async ({teamId, userId, jiraSearchQueries}) => {
```

Comment:
I removed Param type here because `AtlassianAuth` has `jiraSearchQueries` as `JiraSearchQuery` type and newly generated PG methods expect `JsonArray`. We could use `IGetAtlassianAuthByUserIdTeamIdQueryResult` here but I'm not sure if it's the best idea. Any thoughts?